.github/workflows/api-docs.yml

```diff
@@ -48,6 +48,6 @@ jobs:
         if: ${{ steps.docs.outputs.UPDATED_DOCS != 0 }}
         run: |
           git add -u
-          git commit -m 'docs: regenerate'
+          git commit -m 'docs: regenerate [skip ci]'
```

Comment:
This shouldn't be necessary. Workflows can't trigger other workflows under normal circumstances.